package.json

```diff
@@ -66,14 +65,14 @@
         "eslint-plugin-prettier": "^4.0.0",
         "eslint-plugin-react": "^7.30.1",
         "eslint-plugin-react-hooks": "^4.6.0",
-        "faker": "^6.6.6",
```

Comment:
functional code removed  https://www.npmjs.com/package/faker

also not used in this project -> removed from dependencies